openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -436,6 +441,8 @@ static kmp_int32 __kmp_set_stack_info(int gtid, kmp_info_t *th) {
      pthread_attr_getstack may cause thread gtid aliasing */
   if (!KMP_UBER_GTID(gtid)) {
 
+#if !KMP_OS_SOLARIS
+    pthread_attr_t attr;
```

Comment:
I think it would be better to invert the logic and have this section at the top.

Reply:
Also there is a comment above this around line 427. It would probably be best to update that to just 'Set thread stack info.' and leave off the means of doing so now that there are multiple means of doing so.